docs/AOT.md

```diff
@@ -468,6 +468,37 @@ option like in this:
   dot -Tpdf graph.dot -o graph.pdf
   ```
 
+- For debugging purposes you can choose to compile the model in instrumentation
+mode with the option `-instrument-debug` such that the model will display during
+run-time the content of all the tensors associated to all the operands of all the
+IR instructions. Additionally, you can choose the format of the dumped information
+with the option `-instrument-debug-format=<format>`:
+  - `console` (Default) All the operands (tensors) are displayed in text format in the
+  console. Large tensors are only partially displayed.
+  - `bin` The operands (tensors) are dumped in binary format in separate files
+  (one tensor per file). Each file will contain the tensor type and tensor data.
+  - `txt` The operands (tensors) are dumped in text format in separate files
+  (one tensor per file). Each file will contain the tensor type and tensor data. One
+  such file might look like this:
+    ```
+    float<1 x 2 x 3>
+    1.1, 2.2, 3.3, 4.4, 5.5, 6.6,
+    ```
+  - `rawbin` The operands (tensors) are dumped in binary format in separate files
+  (one tensor per file). Each file will contain ONLY the tensor data.
+  - `rawtxt` The operands (tensors) are dumped in text format in separate files
+  (one tensor per file). Each file will contain ONLY the tensor data. One such file
+  might look like this:
+    ```
+    1.1, 2.2, 3.3, 4.4, 5.5, 6.6,
+    ```
+  The names of the dump files have a simple format `data[idx].bin` or `data[idx].txt`
+  but a separate meta file `debug.info` is dumped at compile-time which makes the
```

Comment:
Could you provide in the doc an example of how `debug.info` looks like?

Reply:
Done.